### PR TITLE
chore: simplified the code and removed unnecessary commentary

### PR DESCRIPTION
**Pull request by sayakpaul**

### CODE REVIEW

File: examples/nlp/multi_label_classification.py [hunk 1]
```diff
@@ -370,10 +363,6 @@ def plot_result(item):
 _, categorical_acc = shallow_mlp_model.evaluate(test_dataset)
 print(f"Categorical accuracy on the test set: {round(categorical_acc * 100, 2)}%.")
 
-"""
-The trained model gives us an evaluation accuracy of ~87%.
```
Comment (fchollet) [inline examples/nlp/multi_label_classification.py:374]: This seems like a helpful line, why remove it?

Comment (sayakpaul) [reply examples/nlp/multi_label_classification.py]: Added back in.

**Review by fchollet — APPROVED** [review]
LGTM, thanks! Please update the generated files.

**Comment by sayakpaul** [comment]
@fchollet, thank you! Just added the files.

**Pull request merged**